src/overmap_ui.cpp

```diff
@@ -606,6 +611,7 @@ void draw( const catacurses::window &w, const catacurses::window &wbar, const tr
             const bool los = see && g->u.overmap_los( cur_pos, sight_points );
             const bool los_sky = g->u.overmap_los( cur_pos, sight_points * 2 );
             int mycount = std::count( path_route.begin(), path_route.end(), cur_pos );
+            int player_path_count = std::count( player_path_route.begin(), player_path_route.end(), cur_pos );
```

Comment:
Since you only care whether the count is non-zero you can use `std::find` (with the advantage of early-exit when `cur_pos` is found).